ts/annotations/controllable/ControllableLabel.ts

```diff
@@ -0,0 +1,561 @@
+/* *
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+import H from './../../parts/Globals.js';
+
+/**
+ * Internal types.
+ * @private
+ */
+declare global
+{
+    namespace Highcharts
+    {
+        interface AnnotationAlignObject extends AlignObject {
+            height?: number;
+            width?: number;
+        }
+        class AnnotationControllableLabel implements AnnotationControllable {
```

Comment:
Wouldn't be better to use `extends` ? There will be more classes that can use `AnnotationControllable` base and overwrite it's methods.

Reply:
AnnotationControllable is not a class - just a mixin object - you can not extend a class from an interface.

Reply:
Ah, okay!